internal/install/_static/terraform_deployer_run.sh

```diff
@@ -1,7 +1,9 @@
-#!sh
+#!/usr/bin/env bash
```

Comment:
The reason why the CI is failing is because of the Dockerfile ENTRYPOINT set to `sh`: [here](https://github.com/elastic/elastic-package/blob/main/internal/install/_static/Dockerfile.terraform_deployer#L24). 

Let's quickly refactor it to `/bin/bash` instead and verify.